valohai_yaml/schema/endpoint.yaml

```diff
@@ -41,6 +41,10 @@ properties:
       Specifies the WSGI application to serve (e.g. a Flask application).
       Specify the module (i.e. `package.app`) or
       the module and the WSGI callable (i.e. `package.app:wsgi_callable`).
+  node-selector:
+    type: string
+    description: Kubernetes node selector specifies which nodes the deployment should be scheduled to.
+    pattern: "="
```

Comment:
I considered making it an `object` but opted to go with the string syntax that we use elsewhere (API, UI, admin tooling, etc.)